src/Landing/containers/RecommendArticles.jsx

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Row, Col, Container } from 'react-bootstrap';
+import Article from '../../commons/components/Article';
+import '../CSS/RecommendedArticles.scss';
+import ActionButton from '../components/ActionButton';
+import fetchLatestArticles from '../reduxx/actions/LatestArticles';
+
+class RecommendedArticles extends Component {
+  componentWillMount() {
+    this.props.fetchLatestArticles();
```

Comment:
Shift this block and the store connection to the parent component `Landing` and then pass down the articles as props to `RecommendedArticles`.